ci/deploy_app.sh

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
```

Comment:
The project ID and runner URL needs to be validated to ensure they exist.

Reply:
Added variables validation now, thanks!